packages/cli/src/commands/generate-graphql.ts

```diff
@@ -54,6 +54,16 @@ export default class GenerateGraphql extends Command {
       cwd: isCore ? undefined : tmpDir,
     })
 
+    // yarn generate:copy-back expects the DESTINATION var to be present so it can copy the files to the correct directory
+    runCommandSync({
+      cmd: `DESTINATION=${coreDir} yarn generate:copy-back`,
+      errorMessage:
+        "Failed to format generated files. 'yarn format:generated' thrown errors",
```

Comment:
Can we change the errorMessage to differ from the previous command?

Reply:
Yess! My bad!